tests/golos.publication_rewards_tests.cpp

```diff
@@ -339,6 +360,30 @@ class reward_calcs_tester : public golos_tester {
         }
 #endif
     }
+
+    void set_payout(
+            account_name user, 
+            double payout, 
+            std::string type = "", 
+            double author_token_rwrd = 0.0, 
+            double tokenprop = 0.0) 
```

Comment:
Very strange arguments to function and too difficult to understand what this function do. It's need to refactor this function.

Reply:
@DedbaradeD, looks like here is a bug in additional parameters usage: you already processed author's tokens before call of this function with extended parameters list. so call: https://github.com/GolosChain/golos.contracts/blob/ddcdfd432b879be64535010a22b4ca7207757b3e/tests/golos.publication_rewards_tests.cpp#L447 is for author's vesting part, and there should not be conditions and additional parameters. and this functions should be named "pay_vesting" or so.
to make it more clear, you can add `from` and `to` parameters instead of `user` and hardcoded `_forum_name`.